tt_eager/tensor/tensor.cpp

```diff
@@ -337,9 +337,11 @@ Tensor create_sharded_device_tensor(const Shape& shape, DataType data_type, Layo
 
     ShardSpecBuffer shard_spec_buffer(shard_spec, page_shape, tensor2d_size);
     uint32_t shard_size = shard_shape[0] * shard_shape[1] * element_size;
-    if(layout == Layout::TILE)
+    if(layout == Layout::TILE) {
         shard_size = tensor_impl::packed_buffer_size_bytes_wrapper(data_type, compute_buffer_size(Shape({shard_shape[0], shard_shape[1]}), data_type));
```

Comment:
I don't see this `shard_size` variable used at all anymore in this function? Do we need any of the code related to this?